### ISSUE TITLE
Still cannot figure out how to use setSpellCheckerDictionaryDownloadURL in Electron 8.0.2

### ISSUE THREAD

**Issue by heronhaye** [electron/electron#22482]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * v8.0.2
* **Operating System:**
  * Arch Linux 
* **Last Known Working Electron version:**
  * N/A

### Expected Behavior
Electron to download dictionary files from custom mirror after setting setSpellCheckerDictionaryDownloadURL to the *directory path* containing dictionary files in `hunspell_dictionaries.zip`

### Actual Behavior
It doesn't download the files, and spellchecking doesn't work.

### To Reproduce
Please see sample code at `https://github.com/heronhaye/electron-8-spellchecker-test`. Specifically,

```js
  win.webContents.session.setSpellCheckerDictionaryDownloadURL(
    'https://modalduality.keybase.pub/dictionaries/'
  )
```

### Additional Information
@MarshallOfSound helped me figure out that I was using this function incorrectly before, but now I believe I am using it correctly, but still cannot get it to work. Is there an working example base path somewhere on the internet that I can test it on?

**Comment by MarshallOfSound** [comment]
@heronhaye Hey sorry I was just about to PR up some more documentation today, we're going through the same steps right now so we're finding these things as well.

TLDR is Chromium _might_ request the lowercase version of the dictionary name i.e. `en-us-8...` instead of `en-US-8....`.  You can either:

* Make the file server case insensitive
* Upload the lower case version of each file as well

I'll close this out with a docs PR shortly.

**Comment by MarshallOfSound** [comment]
We're also [adding events](https://github.com/electron/electron/pull/22449) to make these issues easier to debug in the future

**Comment by heronhaye** [comment]
@MarshallOfSound Excellent, I uploaded the lower-case version and it works perfectly. Thank you for the quick responses!
